contributing guide.md

```diff
@@ -0,0 +1,233 @@
+9. Review and Feedback:
+- Be prepared to receive feedback from project maintainers or collaborators. Review and address any comments or suggestions to refine your contribution.
+10. Documentation Completion:
+- After your contribution is reviewed and accepted, your new section will become part of our project's documentation, enhancing its content for users and contributors.
+
+## Using Markdown for Documentation
+Markdown is a lightweight and easy-to-use markup language that allows you to format text in a readable and visually appealing way. When contributing to the OpenSauced project's documentation, it's important to use Markdown to structure and format your content effectively.
+
+Markdown Basics
+Markdown provides a set of simple and intuitive syntax elements for formatting text. Here are some common Markdown elements you can use:
+1. Headings:
+Use `#`  to create headings. The number of '#`  symbols indicates the heading level (e.g.,`# Heading 1`, `## Heading 2` ).
+```bash
+# Heading 1
+## Heading 2
+```
+
+2. Text Formatting:
+- Make text bold by enclosing it with double asterisks (**) or double underscores (__).
+- Make text italic by enclosing it with single asterisks (*) or single underscores (_).
+- Create inline code by wrapping text with backticks (``).
+
+3. Lists:
+- Create ordered lists using numbers followed by a period (1., 2., etc.).
+- Create unordered lists using asterisks (*), plus signs (+), or hyphens (-).
+For instance,
+1. Item 1
+2. Item 2
+
+- Unordered Item 1
+- Unordered Item 2
+
+4. Links:
+- Create links using square brackets for the link text and parentheses for the URL.
+[Link Text](https://opensauced.pizza/)
+
+5. Images:
+- Embed images using an exclamation mark (!), square brackets for the alt text, and parentheses for the image URL.
+![Alt Text](https://github.com/open-sauced/intro/blob/main/opensauced-signup.png)
+
+6. Blockquotes:
+- Create blockquotes using the greater-than symbol (>).
+`> This is a blockquote.`
+
+7. Code Blocks:
+- Create code blocks by indenting code lines with four spaces or using triple backticks for fenced code blocks.
+// Fenced code block
+const code = "Hello, World!";
+
+Markdown Tips:
+- Keep your Markdown content well organized, and use headings to structure your sections.
+- Use code blocks to highlight code snippets or configuration examples.
+- Preview your Markdown locally to ensure proper formatting before submitting your contribution.
+
+## Adding New Sections to the Sidebar
+The sidebar in our project's documentation serves as a navigation menu, allowing users to easily access different sections. When you add a new section to the documentation, it's important to update the sidebar to include a link to the new section. This ensures that users can navigate to your content seamlessly.
+Follow these steps to add a new section to the sidebar:
+1. Locate the Sidebar Configuration:
```

Comment:
Let's move this section to where sidebar is mentioned earlier. Steps:

1. Navigate to `_layouts/sidebar.js`
2. Add New Section link: 
	- To add a new section link, insert a new line with the following format:
	- [Intro](/01-intro.md) -- be sure to capitalize the section title.
	- Replace "Intro" with the title of your new section.
	- Replace "/01-intro.md" with the relative path to the Markdown file of your new section.